core/agent_parser.go

```diff
@@ -116,7 +116,7 @@ func bzeroInit(log logger.T) {
 	//  * Store keys along with passed orgID
 
 	// Safety checks for custom org provider
-	if orgProvider != "google" && orgProvider != "microsoft" {
+	if orgProvider != "google" && orgProvider != "microsoft" && orgProvider != "okta" {
```

Comment:
Do we want to  just remove this block now? I feel like we can clean up the code now that we dont need to support the cisco idp case. Imo having the orgProvider contain the iss url is a hack that we shouldnt try and keep in the code. Lets just make sure every new IdP we support has its own provider enum value and we need to add explicit support for new ones in the agent going forward.